### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by dougwettlaufer**

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+[INSERT CONTACT METHOD].
```
Comment (EricBorczuk) [inline CODE_OF_CONDUCT.md:64]: Should we be inserting this?

**Review by EricBorczuk — APPROVED** [review]

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (dougwettlaufer) [reply CODE_OF_CONDUCT.md:64]: @csplinter ^

Comment (csplinter) [reply CODE_OF_CONDUCT.md:64]: We should, talking to IT about getting a mailing list and @stargateio email set up

Comment (csplinter) [reply CODE_OF_CONDUCT.md:64]: Okay we have an email set up at "maintainers@stargate.io", let's put that in here

**Review by csplinter — APPROVED** [review]

**Pull request merged**